### ISSUE TITLE
Guide for contributors

### ISSUE THREAD

**Issue by daniloraisi** [prisma/prisma-engines#759]
I love what Prisma is doing, changing the way to consume Databases.

I want to contribute with Prisma project, working on MSSQL connector for Prisma Engine, but I wasn't able to found any Documentation regarding guidelines.

It'll be awesome to have some guidance to collaborate on this path.

**Comment by Sytten** [comment]
I believe there is already work done on the mssql with https://github.com/prisma/tiberius
But great point on the contribution documentation, I mostly started by hacking some stuff up.

**Comment by daniloraisi** [comment]
> I believe there is already work done on the mssql with https://github.com/prisma/tiberius
> But great point on the contribution documentation, I mostly started by hacking some stuff up.

This is great! But I think Tiberius is only the driver, right!? I need Prisma Engine Connector to use against MSSQL Databases.

**Comment by Sytten** [comment]
Agreed that will most likely be work done in https://github.com/prisma/quaint

**Comment by daniloraisi** [comment]
> Agreed that will most likely be work done in https://github.com/prisma/quaint

@Sytten what do you think about help me with this?

**Comment by Sytten** [comment]
You should asks the developers before starting working on this, they already have a roadmap. I dont have time personnally.

**Comment by yoshuawuyts** [comment]
Note from triage: we intend to add a CONTRIBUTING.md that will hopefully provide some guidance.

**Comment by yoshuawuyts** [comment]
Closing in favor of https://github.com/prisma/prisma-engines/issues/890
